qcodes/instrument/parameter.py

```diff
@@ -329,7 +330,7 @@ def get_raw(self):
         raise NotImplementedError
 
     @abstractmethod
-    def set_raw(self, value):
+    def set_raw(self, value: Any) -> None:
```

Comment:
I added `ParamRawDataType` in https://github.com/QCoDeS/Qcodes/pull/1757 :)
```suggestion
    def set_raw(self, value: ParamRawDataType) -> None:
```